includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -2221,7 +2212,7 @@ private function handle_saving_payment_method( WC_Order $order, $payment_method_
 	 * @param WC_Order $order The order.
 	 * @param string   $payment_method_id The value to be set.
 	 */
-	private function set_payment_method_id_for_order( WC_Order $order, string $payment_method_id ) {
+	public function set_payment_method_id_for_order( WC_Order $order, string $payment_method_id ) {
```

Comment:
Setting a couple of these functions to public was needed so they could be called within the `confirm_change_payment_from_setup_intent_ajax()` Intent Controller method (which handles the 3DS AJAX request)